charts/rekor/templates/_helpers.tpl

```diff
@@ -460,4 +460,21 @@ Return the location for file based Attestation storage.
 */}}
 {{- define "rekor.server.fileAttestationStorage.path" -}}
 {{- print (substr 7 -1 .Values.server.attestation_storage.bucket) -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Return a random Secret value or the value of an exising Secret key value
+*/}}
+{{- define "rekor.randomSecret" -}}
+{{- $randomSecret := (randAlpha 10) }}
```

Comment:
I wonder if you'd like to use randAlphaNum or even randAscii so as to increase the character set of the secret (are these secrets only meant to use `A-Za-z`?

Reply:
Updated